### ISSUE TITLE
Final Review (sc2886)

### ISSUE THREAD

**Issue by sreenuchandran** [jchen236/genre_classification#14]
The project focuses on classifying songs into a particular genre given details of the song. I think the project is very interesting and relevant as it can greatly improve on the recommendation of songs on music streaming services like Spotify, which I personally use every day.

Things I like about the project:
1) Overall I think it's a really good project. The way the team has gone through the steps are very logical and easily understandable to the reader.
2) The team has tried varied models and also provided conclusions as to why or why not they perform well. It show's they clearly understand the subject well.

Things which can be improved:
1) The team has done a good job in trying various models, but I would have liked to see error metrics for both training and validation sets to better understand the fit. Like for example in section 5.2.1, the team mentions that they found an interesting observation that models performed better when more features were used. But it is not clear if they came across this observation after trying out the model on the test or not since if they measured performance only with training set, it is an obvious result.
2) Since they were getting varied results for each genre, I believe they could have tried using more data points or considereing more sub-genres using a different database since just 13 might not be sufficient. The team has rightly mentiond the latter point in one of their sections too.
3) Could have included a final confidence interval for each model.

Also what if you had tried a PCA model for prediction? Do you think it would have outperformed the other models?

Overall, great work! Was interesting and I had a good time going through it.

**Comment by sz244** [comment]
@sreenuchandran 
Thank you for your comments! We tune our model parameters with our validation set (such as in section 5.2.1) and in the end test our tuned models using test sets. We have used PCA to perform dimensionality reduction(we tried several different dimensions) and performed linear regression on the lower dimensional features. Unfortunately that did not help us improve the model performance. The reason is that I think all of the current features captures some aspect about the song that helps our models make classification decisions.

**Comment by jimmychen623** [comment]
@sreenuchandran Thanks for your feedback! Just wondering, what do you mean when you say "PCA model"?
